packages/grid/data-grid/src/tests/rows.DataGrid.test.tsx

```diff
@@ -57,4 +63,30 @@ describe('<DataGrid /> - Rows', () => {
       expect(getColumnValues()).to.deep.equal(['Mike-11', 'Jack-11', 'Mike-20']);
     });
   });
+
+  it('should ignore events coming from a portal in the cell', () => {
```

Comment:
Not sure we have a test that shows events from a rendercell without portal

Reply:
I don't understand. What do you mean?

Reply:
well you check that the event is blocked with portal. I don't think we have a test that check if the event go through without portal.

Reply:
Ok, I can add this